CarAuction/CarAuction.Data/Interfaces/IAuthRepository.cs

```diff
@@ -7,5 +7,6 @@ public interface IAuthRepository : IRepository
     {
         Task<User> GetByName(string userName);
         Task AddCustomer(User user);
+        bool ExistsLogin(string name);
```

Comment:
```suggestion
        bool CheckIfLoginExists(string name); or bool IsExistingLogin(string name);
```

It is recommended to start the method name with a verb that says what to do (get, check, add, verify etc.)
In case when the return type is bool, the method name starts from "Is" or "Check" usually.